src/Components/Publishing/Sections/StyledText.tsx

```diff
@@ -38,17 +40,30 @@ export const StyledText = div`
   a {
     color: ${props => props.color};
     text-decoration: none;
-    position: relative;
-    background-image: linear-gradient(to bottom,transparent 0, ${props =>
-      props.color === "black" ? "#333" : props.color} 1px,transparent 0);
+    background-image: linear-gradient(
+      to bottom,
+      transparent 0,
+      ${props => (props.color === "black" ? Colors.grayBold : props.color)} 1px,
+        transparent 0
+      );
     background-size: 1.25px 4px;
     background-repeat: repeat-x;
     background-position: bottom;
     &:hover {
-      color: ${props => (props.color === "black" ? "#999" : props.color)};
+      color: ${props =>
+        props.color === "black" ? Colors.grayDark : props.color};
       opacity:  ${props => (props.color === "black" ? "1" : ".65")};
     }
   }
+  ${PrimaryLink} {
```

Comment:
Is that how you're rendering a dotted line? can we keep this styling contained in the `PrimaryLink` styled component instead?

I understand why the `StyledText` component is written that way but it re-introduces some problems you get with CSS where it's hard to track down where some rules are coming from, that styled-components was designed to fix.

Reply:
Yeah I can move it in there, the trade off is we need to use `!important` to override style declarations from the `PrimaryLink` component

Reply:
Also thinking it is time to refactor the `StyledText` component to separate files by layout.